test-error/apps/payslip-validator/src/main/java/com/redhat/training/payslipvalidator/processor/WrongTotalPayslipCalculation.java

```diff
@@ -0,0 +1,4 @@
+package com.redhat.training.payslipvalidator.processor;
+
+public class WrongTotalPayslipCalculation extends Exception {
```

Comment:
Java convention is to end all Exception classes with *Exception - same as Python no?